src/neptune/new/handler.py

```diff
@@ -311,12 +316,13 @@ def log(
                 elif is_float_like(first_value):
                     attr = FloatSeries(self._container, parse_path(self._path))
                 elif is_string_like(first_value):
-                    warn_once(
-                        message="The object you're logging will be implicitly cast to a string."
-                        " We'll end support of this behavior in `neptune-client==1.0.0`."
-                        " To log the object as a string, use `.log(str(object))` instead.",
-                        stack_level=3,
-                    )
+                    if not from_stringify_value:
```

Comment:
This is for `log()`; what about `append()` and `extend()`?

Reply:
The same message is for both `append` and `extend` and it's located in `type_casting.py`

Reply:
I cannot find it. I was just wondering since the text mentions `log()` but we generally want to use `append()` instead, since we no longer use `log()` in docs or examples.

Reply:
`cast_value_for_extend` in t`type_casting.py:100`. As it might be called with `append` and `expand` it wasn't mentioned (or probably should take into account the "or" operator).

About `log` -> I know, I know, but still, it wasn't deprecated ("yet") and `extend` might not be as common for users as we may think and the message that uses `extend` when `log` was originally called might be misleading.

Reply:
Got it, all good